src/server/storiesProviders/storiesProviderFactory.ts

```diff
@@ -0,0 +1,6 @@
+import { StoriesProviderFactory } from '../../types';
+import { createLocalWebpackStoriesProvider } from './localWebpackStoriesProvider';
+
+export const storiesProviderFactory: StoriesProviderFactory = (options) => {
```

Comment:
This function will resolve right StoriesProvider implementation and create it.

For now it just creates the only one current implementation